### PR TITLE
TralezaboOoOoo Part 1/1: Paranormal Mulebot-chan Carries Your Ghosts

### PR DESCRIPTION
**Pull request by tralezab**
~~MUST be merged AFTER 45943~~ That's merged!

## About The Pull Request

I'm still deciding where this should be obtained, maybe a ruin? maybe just admin??

Unlike the mulebot that only carries crates unless you hack it...
[this is a mulebot that only carries ghosts unless you hack it!](https://youtu.be/E84tXgnh5QI)

## Why It's Good For The Game

A lot of people really like the ghostly interactions with the mulebot right now and I thought this would be a really fun way to replace it in homage. It's also really fucking funny to run people over with a vehicle carrying a ghost

## Changelog
:cl: Fikou for sprites
add: Paranormal mulebot. It can't even carry anything, this is useless garbage!
/:cl:

### CODE REVIEW

**Comment by tralezab** [comment]
sprites of an old sparking mulebot if this survives

**Comment by PKPenguin321** [comment]
Epic PR title

**Comment by tralezab** [comment]
video added for your viewing pleasure

**Comment by tralezab** [comment]
thanks fikou for sprites!
image

**Comment by 81Denton** [comment]
Charon, the ~~ferryman~~ mulebot carrying the souls of the deceased across the ~~river Styx~~ cargo bay.

**Comment by ExcessiveUseOfCobblestone** [comment]
I just realized I spelt caduceus wrong while already intentionally butchering the word to shill my username so  I see where you stand.
 
Might I recommend "Tralezabargo" since this is a cargo oriented PR?

**Comment by tralezab** [comment]
Normally I wouldn't bug maintainers more buuut it has in fact been bugging me that I didn't capitalize "carries your ghosts"

File: code/modules/mob/living/simple_animal/bot/mulebot.dm [hunk 1]
```diff
@@ -739,6 +743,91 @@
...
+	update_icon()
+	if(istype(AM, /mob/dead/observer))
+		ghost_rider = TRUE
+		RegisterSignal(AM, COMSIG_MOVABLE_MOVED, .proc/ghostmoved)
+
+/mob/living/simple_animal/bot/mulebot/paranormal/update_icon()
+	if(load && isobserver(load) && isnull(ghost_overlay))//there are issues with adding a ghost as an overlay, and this prevents metagaming to see who is dead
+		ghost_rider = TRUE
+		visible_message("<span class='warning'>A ghostly figure appears on [src]!</span>")
+		ghost_overlay = ghost_overlay || mutable_appearance('icons/mob/mob.dmi')
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/mob/living/simple_animal/bot/mulebot.dm]: braintoot: does this not return a T/F?

Comment (tralezab) [reply code/modules/mob/living/simple_animal/bot/mulebot.dm]: I actually have no idea how mutable appearances work....

Comment (ExcessiveUseOfCobblestone) [reply code/modules/mob/living/simple_animal/bot/mulebot.dm]: this is fine, i just braintooted

Comment (ShizCalev) [inline code/modules/mob/living/simple_animal/bot/mulebot.dm:805]: working on some shit atm and came across this

```
if(isnull(ghost_overlay))

ghost_overlay = ghost_overlay
```

?????????

**Pull request merged**